wiki/Ranking_Criteria/osu!catch/zh.md

```diff
@@ -1,193 +1,187 @@
----
-outdated: true
-outdated_since: 0c67d13667b2e598e463c9a46234f929669d9d2d
----
-
 # osu!catch ranking criteria
 
-**osu!catch Ranking Criteria** 是一系列关于创建 osu!catch 模式谱面的规定和指南。为了使 osu!catch 模式谱面被 Rank，谱面必须遵循以下列出的标准。**规定在任何情况下都须遵守**，而指南可以在某些情况下打破。但这些例外的情况必须有详尽合理的解释，解释指南被打破的原因，以及为什么你认为遵守指南会影响谱面的整体质量。
+***注意: 这个文件是[通用 Ranking Criteria](/wiki/Ranking_Criteria)的扩展。***
 
-## 术语
+为了使 [osu!catch](/wiki/Game_mode/osu!catch) 特定模式的[谱面](/wiki/Beatmap)通过[谱面 Ranking 程序](/wiki/Beatmap_ranking_procedure)，则必须遵守 **osu!catch Ranking Criteria** 内列出的[规定和准则](/wiki/Ranking_Criteria#general-terms)。
 
-这些术语常见于与 osu!catch 难度相关的地方，也在 Ranking Criteria 中使用。
+## 全局
 
-### 难度名
+全局规定和准则适用于所有种类的 osu!catch 难度。与节奏相关的规定和准则适用于歌曲为 4/4 拍，约 180 BPM 的谱面。如果歌曲明显快于或慢于 180 BPM，某些规则内限制的指标会不一样。更详细的说明请参考：[缩放 BPM](/wiki/Ranking_Criteria/Scaling_BPM)
 
-*主页面：[难度命名](/wiki/Ranking_Criteria/Difficulty_naming)*
+### 常规
 
-- ![](/wiki/shared/diff/easy-c.png?20211215) Cup
-- ![](/wiki/shared/diff/normal-c.png?20211215) Salad
-- ![](/wiki/shared/diff/hard-c.png?20211215) Platter
-- ![](/wiki/shared/diff/insane-c.png?20211215) Rain
-- ![](/wiki/shared/diff/expert-c.png?20211215) Overdose
-
-### 游戏术语
+#### 规定
 
-- **水果（Fruit）：** 一个可以代表圆圈、滑条头、滑条尾或者折返的大物件。
-- **水滴（Drop）：** 一个代表滑条节奏点的中等大小物件。
-- **小水滴（Droplet）：** 一个代表滑条体的小物件。没接住它们会降低你的准确度，但是与前面两种物件不同的是，它不会造成连击中断。
-- **香蕉（Banana）：** 一个在转盘时出现的物件。它们有额外得分，但是不会影响准确度和连击。
-- **有效物件（Active object）：** 为达到全连所必须接住的物件，也就是水果和水滴。
-- **走（Walk）：** 不使用跳就能接住的物件间距。
-- **跳（Dash）：** 需要按住加速键来接住两个物件的间距。
-- **红果跳（Hyperdash）：** 两个有效物件间间距太大，只按住加速键是接不到的。在游戏中，会由前一个物件发光来表示。也用来指这两个物件之间的距离。
-- **触发距离（Trigger distance）：** 两个有效物件之间产生红果跳所需的最短距离。
-- **边缘跳（Edge dash）：** 当两个有效物件之间距离较大但又未达到触发距离时，第一个物件必须用盘子的边缘去接才能接住第二个物件。
-- **基础跳/红果跳（Basic dash/hyperdash）：** 两个有效物件间时间间隔在允许跳的最小时间两倍或以上时的跳或红果跳。例如，Platter 难度中时间间隔在 250 毫秒的跳是一个基础跳。
-- **短时跳/红果跳（Higher-snapped dash/hyperdash）：** 时间短于最小允许时间间隔两倍的跳或红果跳。
-- **反向梗（Antiflow）：** 与自然移动相比强烈的方向或速度变化。
+- **你的谱面必须理论上可以 SS。** 这意味着必须可以完全接住所有[水果/大果](/wiki/Hit_object/Fruit)、[水滴/中果](/wiki/Hit_object/Juice_stream#drop)、[小水滴/小果](/wiki/Hit_object/Juice_stream#droplet)。
```

Comment:
this may be a standard format and it's ok to keep it

Reply:
so it's standard to have a space after the `。`? that's not the case anywhere else. if this isn't an exception in the language, then there shouldn't be a space.

Reply:
thx for the review!
about space problem, here is a example:
**加粗文字。**非加粗文字  
**加粗文字。** 非加粗文字
````
**加粗文字。**非加粗文字  
**加粗文字。** 非加粗文字
````
as you can see, if no space between the bold marker and the Chinese character, the bold doesn't work.
also, the English version of this article also has such space, see below:
![image](https://user-images.githubusercontent.com/20500319/154480414-9f368e38-e60b-4483-87cf-145de0ec80b1.png)

Reply:
it may be that github can't display it properly. have you tried this before and checked on the osu website, with it not working? the bolding should work regardless of whether there's a space. if not, then it's a bug that should be reported in osu-web.

in english, there is a space after the period because that's in the punctuation rules. i know japanese uses the same `。` character and doesn't use a space after, and i suspect it's the same in chinese based on observation. the space should therefore be removed.

Reply:
Hmm sounds reasonable, I will try to find some examples on osu!web later, may take some hours since busy IRL. I will replace once confirmed that's true.

Reply:
Just going to note that this was [discussed on discord](https://discord.com/channels/188630481301012481/218677502141399041/944338079668387932) and both the [commonmark reference implementation](https://spec.commonmark.org/dingus/) and [the spec](https://spec.commonmark.org/0.30/#emphasis-and-strong-emphasis) corroborate that the space (or an alternative whitespace character) is needed there to make work.

A similar ASCII-only example of the same principle would be the markdown string of `*test.*test` - the second asterisk doesn't work as a termination because it's not followed by a whitespace-class Unicode codepoint.